WordPress/metadata/PlayStoreStrings.po

```diff
@@ -11,6 +11,18 @@ msgstr ""
 "Project-Id-Version: Release Notes & Play Store Descriptions\n"
 
 #. translators: Release notes for this version to be displayed in the Play Store. Limit to 500 characters including spaces and commas!
+msgctxt "release_note_165"
+msgid ""
+"16.5:\n"
+"Blocks: We added cross-post support for P2, full-width and wide alignment support in the Columns block, the ability to pick a link in the Image block, and the ability to move a block by long-pressing the up/down buttons.\n"
+"\n"
+"Activity Log: Track your site activity with filters that sort by date or type.\n"
+"\n"
+"Site Pages: Copy a page with the Duplicate button.\n"
+"\n"
+"\n"
+msgstr ""
+
```

Comment:
Is it expected that we _don't_ have short release notes for 16.5 (i.e. a new entry for `release_note_short_165` key) as we did for e.g. 16.3?

Do we only add that entry if the full release notes looks too close to the 500 chars limit and Editorial provided an alternate shorter version and that's why we don't have any this time around?

Reply:
Yep, this is right!